spec/system/placements/start_page_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rails_helper"
+
+RSpec.describe "Start Page", type: :system, service: :placements do
```

Comment:
Could you rename this to `landing_page_spec.rb`?

```suggestion
RSpec.describe "Landing Page", type: :system, service: :placements do
```